frontend/src/Css.tsx

```diff
@@ -46,6 +46,7 @@ export const color = {
   infoBg: '#f3f4ff',
   infoText: '#1a73e8',
   weak: '#9aa0a6',
+  link: '#00458b',
```

Comment:
Thanks for the try! It does feel better to me from previous revision

nit we might consider using material design color tool to pick the darker main theme color:
https://material.io/resources/color/#!/?view.left=0&view.right=0&primary.color=1a73e8

#0049b5

so that the color is not just for link, it's part of the theme palette and just a darker version of the main color, whenever we need it.

Reply:
Sounds good, thank you for the reference to palette tool! I am going to pick color from there.